### PR TITLE
Feature/issues/6

### PR DESCRIPTION
**Pull request by seanl-adg**

### CODE REVIEW

File: src/components/ProgressBar.js [hunk 1]
```diff
@@ -1,13 +1,19 @@
...
-    }
-    render() {
-        return (
-            <div>
+
+function ProgressBar(props) {
+    return (
+        <div>
+            <div className="progress">
+                <div className="progress-bar" role="progressbar" style={{'width': String(props.currentPage / 7 * 100) + '%'}}/>
```
Comment (sc0rp10) [inline src/components/ProgressBar.js]: why 7?

File: src/github/parse.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+function tryParsingFromTitle(title) {
+    let ar = title.split(/[\s\-]/);
+    let candidates = getCandidates(ar);
+    return candidates;
+}
+
+function tryParsingFromBody(body) {
+    body = body.slice(body.indexOf('***Site***') + 10);
```
Comment (sc0rp10) [inline src/github/parse.js]: what mean 10?

File: src/github/parse.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+
+function parseDesc(issue) {
+    let body = issue.body;
+    body = body.replace(/\[\/\/\][\s\S]*?\r\n/g, ""); // remove invisible lines
+    body = body.trim();
+    let i = body.indexOf("***Description***");
+    if(i != -1) {
+        body = body.slice(i + 37);
```
Comment (sc0rp10) [inline src/github/parse.js]: what mean 37?

File: src/github/parse.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+        body = body.slice(0, body.indexOf("**"));
+    }
+    else {
+        i = body.indexOf("***");
+        if(i != -1) {
+            body = body.slice(0, i);
+        }
+    }
+    return body.slice(0, 40);
```
Comment (sc0rp10) [inline src/github/parse.js]: what mean 40?

File: src/utils/parse-url.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+module.exports = {
+    extractDomain: function(url) {
+        let match = R_URL_DOMAIN_MATCH.exec(url);
+        if(match && match[1]) {
+            return match[1].replace(/\.$/g, '');
+        }
+        else return false;
+    }
+};
```
Comment (sc0rp10) [inline src/utils/parse-url.js]: please add a newline to the EOF

File: src/components/pages/ProbURL.js [hunk 6]
```diff
@@ -92,6 +97,71 @@ AppDetails = connect((state) => ({
     problemURL: state.problemURL
 }))(AppDetails);
 
+class RelatedIssues extends React.Component {
+    constructor(props) {
+        super(props);
+        this.componentDidMount = this.componentDidMount.bind(this);
```
Comment (sc0rp10) [inline src/components/pages/ProbURL.js]: why do you save internal react methods to local links?

File: src/components/pages/ProbURL.js [hunk 7]
```diff
@@ -92,6 +97,71 @@ AppDetails = connect((state) => ({
...
+        this.componentWillReceiveProps = this.componentWillReceiveProps.bind(this);
```
Comment (sc0rp10) [inline src/components/pages/ProbURL.js]: again

File: .gitignore [hunk 8]
```diff
@@ -41,3 +41,6 @@ credentials
 
 #issues database
 issues.db
+
+#yarn
+yarn.lock
```
Comment (TPS) [inline .gitignore]: Specifically, https://github.com/yarnpkg/yarn/issues/1583#issuecomment-294062586 (primarily) & https://github.com/yarnpkg/yarn/issues/1583#issuecomment-279409311, which each say https://github.com/yarnpkg/yarn/issues/1583#issuecomment-257544217 is wrong under different circumstances.

**Pull request merged**